assets/js/langpack/es.json

```diff
@@ -104,7 +104,7 @@
     "Hardware",
 
     "Meet up with openSUSE at conferences.":
-    "Encuéntrate con openSUSE en conferencias.",
+    "Encuéntrate con openSUSE en las conferencias.",
```

Comment:
Aquí creo que debería permanecer sin el artículo dado que en ingles tampoco lo han redactado como "at the conferences" porque el artículo denota una conferencia en particular, cuando en realidad se habla de conferencias en general :)